gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.11.1-bin.zip
```

Comment:
question: have you executed `./gradlew wrapper --gradle-version=8.11.1`? To update the wrapper and scripts